### DISCUSSION TITLE
Retrospective sprint 0

### DISCUSSION THREAD

**Discussion by nadires** [pgCatamarca2021fs/aula1-grupoa-team-1a#45] [General]
Hola chicos, cuando puedan vayan dejando una opinión por cada item así vamos avanzando con la retrospective 
1-¿Qué cosas han funcionado bien?

2-¿Cuales hay que mejorar?

3-¿Qué cosas quiere probar hacer en la siguiente iteración?

4-¿Qué hemos aprendido?

5-¿Cuales son los problemas que podrían impedirle progresar adecuadamente? 

@jorgea306 @PaolaTapia

**Comment by PaolaTapia** [comment]
1. La asignación de tareas. Volver a las reuniones para decidir quien hace cada parte.  
2. Hay que mejorar la integración. Ver que todo quede integrado. Y asegurarnos que todos partimos del mismo punto en el siguiente sprint.  Hacer un pull con todo lo que se subió y fusionó.  
3. Hemos aprendido, git, html, css, bootstrap.
4: Podría impedir avanzar adecuadamente, que no estén bien detallas y separadas las tareas.

**Comment by jorgea306** [comment]
1-¿Qué cosas han funcionado bien?
Los meet que teníamos periódicamente 

2-¿Cuales hay que mejorar?
La organización en general

3-¿Qué cosas quiere probar hacer en la siguiente iteración?
Lo aprendido en github

4-¿Qué hemos aprendido?
Hemos aprendido, git, html, css, bootstrap.

5-¿Cuales son los problemas que podrían impedirle progresar adecuadamente?
La falta de comunicación y coordinación de las actividades

**Comment by GuillermoAVelasco** [comment]
1-¿Qué cosas han funcionado bien?

A mi parecer todo a funcionado bastante bien, el grupo con quien me tocó trabajar es excelente, se desempeñan bien, son colaborativos y responsables.

2-¿Cuales hay que mejorar?

Y se puede ir mejorando los tiempos de avances.

3-¿Qué cosas quiere probar hacer en la siguiente iteración?

implementar mas Scrum.

4-¿Qué hemos aprendido?

Scrum.

5-¿Cuales son los problemas que podrían impedirle progresar adecuadamente?

La desmotivación y cansancio de los integrantes del equipo por los tiempos que cada uno posee.

**Comment by IZI003** [comment]
1-¿Qué cosas han funcionado bien?
Los meet funcionaron muy bien. 
2-¿Cuales hay que mejorar?
Necesitamos tener nuevamente los objetivos claros y cortos.
3-¿Qué cosas quiere probar hacer en la siguiente iteración?
quiero comenzar a programar. al final lo que menos hemos hecho en 2 o mas meses es programar.
4-¿Qué hemos aprendido?
trabajar con un grupo que todos estaban en el mismo nivel, genero mas interés por aprender ya que surgían muchas preguntas en cuanto a scrum.
5-¿Cuales son los problemas que podrían impedirle progresar adecuadamente?
en este periodo las vacaciones, estudios por exámenes, enfermedad. crean un ambiente que baja los ánimos porque nadie esta disponible.
